### PR TITLE
add jgrafzahl application

### PR DESCRIPTION
**Pull request by elmiko**

### CODE REVIEW

**Comment by elmiko** [comment]
@mattf ptal

**Comment by elmiko** [comment]
@crobby ptal

File: _applications/jgrafzahl.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+count. The word fountain uses the `SERVERS` environment variable to
+find the Apache Kafka deployment to use. In the second step, when you
+created `apache-kafka` you created a service with the same name on
+port 9092. Note: The first time this step and the next run you'll have
+to wait for the builder images to be pulled down from the internet, so
+if you're on a thin pipe you may want to start both at the same time
+and grab a drink.
+
+```
+oc new-app openshift/python-27-centos7~https://github.com/mattf/word-fountain -e SERVERS=apache-kafka:9092
```
Comment (crobby) [inline _applications/jgrafzahl.md]: Do we want to point at matt's github?  shouldn't we host it?

File: _applications/jgrafzahl.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+```
+
+Fourth, launch jGraf Zahl herself, using the Oshinko java S2I
+builder.
+
+```
+oc new-app --template=oshinko-java-spark-build-dc \
+           -p APPLICATION_NAME=jgrafzahl \
+           -p GIT_URI=https://github.com/elmiko/jgrafzahl \
```
Comment (crobby) [inline _applications/jgrafzahl.md]: same here...should we avoid our own githubs?

File: _applications/jgrafzahl.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+First, make sure you are connected to an OpenShift cluster and are in
+a project with Oshinko installed. See [Get Started](/get-started) if
+you need help.
+
+Second, load the Apache Kafka infrastructure components into your
+project and start them.
+
+```
+oc create -f https://raw.githubusercontent.com/mattf/openshift-kafka/master/resources.yaml
```
Comment (crobby) [inline _applications/jgrafzahl.md]: another mattf reference

File: _applications/jgrafzahl.md (hunk 2 above)
Comment (elmiko) [inline _applications/jgrafzahl.md]: in general i would like to see more stuff getting migrated to the radanalytics org, matt has those links to word-fountain and openshift-kafka in the grafzahl tutorial as well. i'm not sure if they will be migrated to radanaytics, but i can certainly move jgrafzahl over.

good spot, and i think we need a stronger policy about this going forward.

**Comment by elmiko** [comment]
@crobby ok, i changed the jgrafzahl link and moved the project to radanalyticsio

**Comment by elmiko** [comment]
> I'd rather see us referencing the radanalytics githubs for the projects to promote stability.

(somehow my previous comment got deleted)

i agree, i'm not sure what the larger story about the openshift-kafka and word-fountain repos is. i know matt just migrated the grafzahl repo to radanalyticsio, and i've done the same with jgrafzahl. for now, those links follow the same pattern that's in the grafzahl example.

i think we need some sort of policy about this going forward, but we'll have to discuss it to see what works best.

**Pull request merged**